release-notes/azure/v11.1.0.md

```diff
@@ -0,0 +1,47 @@
+## :zap: Giant Swarm Release 11.1.0 for Azure is now active for you! :zap:
+
+This release introduces support for multiple Availability Zones, as well as graceful upgrades, and improved worker labeling.
+
+### Important upgrade notes
+- Old clusters can not be upgraded to support multiple Availability Zones. In order for a cluster to be able to scale across multiple Availability Zones, it must be created using the v11.1.0 release.
```

Comment:
should we clarify that this is due to Azure limitations?

Reply:
Yeah, that sounds good 👍 I'll do it now